src/Controls/src/Core/Application/Application.cs

```diff
@@ -176,6 +176,7 @@ public AppTheme UserAppTheme
 				_userAppTheme = value;
 
 				TriggerThemeChangedActual();
+				OnPropertyChanged();
```

Comment:
I wonder what this does to the performance because this will trigger a lot of ui elements potentially? It looks like `TriggerThemeChangedActual` tries to do some smart things to see if we actually need to trigger. Should we do that here too?

Reply:
Hmm, that's a good point. I've pushed another commit